scout/constants/variant_tags.py

```diff
@@ -96,6 +98,20 @@
     ("XD_dn", "X Linked Dominant De Novo"),
 )
 
+# Associates an inheritance model with a color, using the same colors categories as scout.constants.gene_tags.INHERITANCE_PALETTE
+# The colors of this palette are the same as the MatPlotLib Tableau palette (https://matplotlib.org/3.4.1/gallery/color/named_colors.html)
+GENETIC_MODELS_PALETTE = {
+    "AR_hom": INHERITANCE_PALETTE["AR"],
```

Comment:
Here I think we will want a graded intensity. AR should be more visible than AR_comp (which anyway has a Compund button connected, and will be nice with a thematic color), AD_dn more intense than AD etc.